pulpcore/tests/functional/__init__.py

```diff
@@ -763,14 +763,15 @@ def __exit__(self, exc_type, exc_value, traceback):
 def random_artifact_factory(
     artifacts_api_client, tmp_path, gen_object_with_cleanup, pulp_domain_enabled
 ):
-    def _random_artifact_factory(pulp_domain=None):
+    def _random_artifact_factory(size=None, pulp_domain=None):
```

Comment:
```suggestion
    def _random_artifact_factory(size=32, pulp_domain=None):
```
We don't rely on writing a uuid there...